src/main/java/frc/robot/Robot.java

```diff
@@ -65,4 +88,21 @@ public void teleopInit() {
             autonomousCommand.cancel();
         }
     }
+
+    public Command doANetworkTableThing() {
```

Comment:
When is this command used?

Reply:
I have a better question: what does it do?

Reply:
·